hoomd/md/methods.py

```
@@ -619,17 +604,13 @@ def thermalize_barostat_dof(self, seed):
             prepare a newly created `Simulation`.
 
         .. seealso:: `State.thermalize_particle_momenta`
-
-        Note:
-            The seed for the pseudorandom number stream includes the
-            simulation timestep and the provided *seed*.
         """
         if not self._attached:
             raise RuntimeError("Call Simulation.run(0) before"
                                "thermalize_thermostat_and_barostat_dof")
 
         self._cpp_obj.thermalizeThermostatAndBarostatDOF(
```

We should raise a warning here if the seed is unset.